client/src/components/Articles/detailArticle.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+<div>
+
+    <div class="mb-4 " v-for="(article, i) in details" :key="i">
```

Comment:
ini juga jangan di v-for disini.. sebaiknya pas klik readMore itu km kirim parameter si article detail nya, jadi disini tinggal ngurusin display nya aja